README.md

```diff
@@ -173,6 +173,12 @@ project-specific guides take precedence for that project.
   "string#{expr}"
   ```
 
+This applies to HAML too:
+
+```Haml
+  .div{ class: computed_classes }
+```
+
```

Comment:
We're using `haml-lint` in PT Hub, that's very useful!
* https://github.com/net-engine/pt_hub/blob/master/Gemfile#L41
* https://github.com/net-engine/pt_hub/blob/master/.travis.yml#L25